### DISCUSSION TITLE
Compile assets after clean

### DISCUSSION THREAD

**Discussion by idc77** [mastodon/mastodon#29274] [Q&A]
I changed some language files' contents.
Did a `RAILS_ENV=production bundle exec rails assets:clean`
and want to recompile the assets again
but

```
RAILS_ENV=production bundle exec rails assets:precompile
Everything's up-to-date. Nothing to do
```

How can I compile the assets?

**Comment by Tslmuun** [comment]
Please clean up frontend assets and recompile.
For example: `rm -rf /home/mastodon/live/public/packs/*`
Then, run `RAILS_ENV=production bundle exec rails assets:precompile` again.
